.github/workflows/build.yml

```diff
@@ -20,7 +20,8 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          ref: ${{ github.head_ref }}
+          ref: ${{github.event.pull_request.head.ref}}
```

Comment:
My fork does not checkout without these changes.

Reply:
That's on purpose actually.  Because we accept contributions from those outside of our organization we don't want this change included.

Reply:
Here is the error I was getting prior to making the change above.
https://github.com/im-open/git-version-lite/actions/runs/4049349922/jobs/6965611905

![image](https://user-images.githubusercontent.com/8923247/215896101-09971833-eeb3-460b-8213-619507adf610.png)

I don't have permissions to see what roles I have been assigned on this repo.  I will need someone else to verify why this is happening.

Reply:
I created my fork through the GitHub UI as well.

> When I created my PR I had the box checked that granted the maintainers access.
I don't know what you mean by that? Perhaps you can demonstrate with a screenshot?

Reply:
I get the same result when attempting to fork and build a PR on other repos in this org of which I am not a contributor.